mongo/change_stream.go

```diff
@@ -343,7 +343,7 @@ func (cs *ChangeStream) buildPipelineSlice(pipeline interface{}) error {
 
 	for i := 0; i < val.Len(); i++ {
 		var elem []byte
-		elem, cs.err = transformBsoncoreDocument(cs.registry, val.Index(i).Interface())
+		elem, cs.err = transformBsoncoreDocument(cs.registry, val.Index(i).Interface(), true, "pipeline")
```

Comment:
The parameter name should include the index of the pipeline stage. Otherwise, a user may think we're saying that the entire pipeline is invalid.